Documentation/user-manual.txt

```diff
@@ -374,6 +374,13 @@ of the dive computer (at least for those not charging while connected via USB).
  - Do *not* check the checkboxes labelled _Save libdivecomputer logfile_ and
    _Save libdivecomputer dumpfile_. These are only used as diagnostic tools
    when there are problems with downloads(see below).
+
+ - With some dive computers it is possible to adjust the clock on the dive
```

Comment:
it would be nice to give a few examples here... which dive computers support this today?

Reply:
```
~/git/subsurface$ grep -l _device_timesync libdivecomputer/src/*.c
libdivecomputer/src/deepblu_cosmiq.c
libdivecomputer/src/deepsix_excursion.c
libdivecomputer/src/device.c
libdivecomputer/src/divesystem_idive.c
libdivecomputer/src/hw_frog.c
libdivecomputer/src/hw_ostc3.c
libdivecomputer/src/hw_ostc.c
libdivecomputer/src/mclean_extreme.c
libdivecomputer/src/oceans_s1.c
libdivecomputer/src/sporasub_sp2.c
libdivecomputer/src/suunto_eonsteel.c
```

These are the dive computer models that libdivecomputer is currently implementing time sync for - with the hope that the number will raise over time. I have started looking into implementing it for additional models, as it is a pain to have to use a manufacturer (or a minimal button UI) just to get the correct dive time logged when travelling.

I haven't added a list of models to the manual as I really see this as a snapshot of current libdivecomputer's capabilities, which likely will become stale before too long as nobody will update it when new models are added to libdivecomputer.